flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -2416,5 +2416,8 @@
     },
     "dev.neovide.Neovide": {
         "finish-args-flatpak-spawn-access": "required to allow spawning neovim on the host"
+    },
+    "com.core447.StreamController": {
+        "finish-args-flatpak-spawn-access": "required to allow plugins to open apps on the host"
```

Comment:
Where can I find an example of this type of plugin? The plugin repo you pointed to in submission seems to have nothing

Reply:
The mentioned OS plugin can be found [here](https://github.com/StreamController/OSPlugin) and [this](https://github.com/StreamController/OSPlugin/blob/main/actions/RunCommand/RunCommand.py) is the action that needs the spawn permission.